.editorconfig

```diff
@@ -185,9 +191,21 @@ dotnet_diagnostic.CA2200.severity = warning
 # CA2208: Instantiate argument exceptions correctly
 dotnet_diagnostic.CA2208.severity = warning
 
-[**/HostFactoryResolver.cs]
-# CA1823: Use literals where appropriate
-dotnet_diagnostic.CA1802.severity = suggestion
+# IDE0035: Remove unreachable code
+dotnet_diagnostic.IDE0035.severity = warning
+
+# IDE0036: Order modifiers
+dotnet_diagnostic.IDE0036.severity = warning
+
+# IDE0043: Format string contains invalid placeholder
+dotnet_diagnostic.IDE0043.severity = warning
+
+# IDE0044: Make field readonly
+dotnet_diagnostic.IDE0044.severity = warning
+
+# IDE0073: File header
+dotnet_diagnostic.IDE0073.severity = warning
+file_header_template = Copyright (c) .NET Foundation. All rights reserved.\nLicensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
```

Comment:
@pranavkm There is some shared code from runtime that doesn't follow this standard. e.g. https://github.com/dotnet/aspnetcore/blob/348b810d286fd2258aa763d6eda667a83ff972dc/src/Shared/runtime/Http3/QPack/HeaderField.cs

Is there a way to disable the rule with some files?

Reply:
Something like this would work: https://github.com/dotnet/aspnetcore/blob/98a8d7a2be94f04f65e3463ab897fde58a62edb1/.editorconfig#L188-L190